max-severyn/lab-4/User.py

```diff
@@ -0,0 +1,28 @@
+from BankUser import BankUser
+from FileManagement import FileManagement
+
+
+class User(BankUser):

```

Comment:
`BankUser` should inherit from `User`, not vice versa. `User` is more general and only implements create/delete operations.
Then `BankUser` should simply refer to the `super.__init__` constructor.

Reply:
I did a mistake naming these 2 classes as BankUser and User. They should be treated as BankAccount and AccountOperations.